hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -387,15 +388,20 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
   );
 
   handleExistsAtChange = (e: any) => {
-    console.log("handleExistsAtChange", e.key);
-    if (e.key === 'OM') {
-      mismatchPrevKeyList = [0];
-      this.fetchMismatchContainers(this.state.DEFAULT_LIMIT, 0, 'SCM');
-    }
-    else {
-      mismatchPrevKeyList = [0];
-      this.fetchMismatchContainers(this.state.DEFAULT_LIMIT, 0, 'OM');
-    }
+    this.setState({
+      pageDisplayCount: 1
+    }, () => {
+      if (e.key === 'OM') {
+        //mismatchPrevKeyList = [0];
+        prevKeyListMap.clear();
+        this.fetchMismatchContainers(this.state.DEFAULT_LIMIT, 0, 'SCM');
```

Comment:
On line 394 we are checking if the key is equal to `'OM'`, so shouldn't we fetch with passing `'OM'` here?

Reply:
Keys = om then we are calling API with  missingIn =SCM means records for OM.